main.py

```diff
@@ -70,7 +70,7 @@ def get_formations(record):
         age = response['age_description'][0]
 
         for usage in response['usages']:
-            usages = [usage['usage']]
+            usages = usage['usage']
 
```

Comment:
@ben-norton  Why are you using  the last usage, but grabbing the first state and lithology?